cypress/integration/Signup/neetoAuthSignup.spec.js

```diff
@@ -0,0 +1,107 @@
+ /// <reference types="cypress" />
+
+import { sigupSelectors,  infoDetailsSelectors, OTPSelectors, passwordSelector, organisationSelector, cancelSelectors, profileDashboard, messageSelector } from '../../constants/selectors/selectors'
+import { texts } from '../../constants/texts/texts'
+import { routes } from '../../constants/routes/routes'
+import { formFill } from '../../utils/formFill';
+const faker = require('faker');
```

Comment:
Please import external files before internal files. Please check guidelines. Also please store selectors in proper variables, we don't need so many variables.
```suggestion
import faker from "faker";

import { 
  sigupSelectors,  
  infoDetailsSelectors,
  OTPSelectors, 
  passwordSelector, 
  organisationSelector, 
  cancelSelectors, 
  profileDashboard, 
  messageSelector }
from '../../constants/selectors/selectors'
import { texts } from '../../constants/texts/texts'
import { routes } from '../../constants/routes/routes'
import { formFill } from '../../utils/formFill';
```